stagemonitor-core/src/main/java/org/stagemonitor/core/util/HttpClient.java

```diff
@@ -81,6 +79,7 @@ public int send(final String method, final String url, final Map<String, String>
 
 		HttpURLConnection connection = null;
 		InputStream inputStream = null;
+		String urlWithoutAuthInfo = removeUserInfo(url);
```

Comment:
inline variable. if there are problems with the removeUserInfo method it limits the scope